### ISSUE TITLE
cdbg_extra_class_path read before Tomcat deploys the war file.

### ISSUE THREAD

**Issue by yuvaldrori** [GoogleCloudPlatform/cloud-debug-java#14]
The documentation describes the way to add extra Classpath for Tomcat applications. Following the instruction I would get an "File was not found in the executable" error from the Stackdriver UI. It looks like the agent loads the extra ClassPath files before Tomcat deployed the war file.

**Comment by hrasadi** [comment]
When do you get the error message? While setting a breakpoint? By default, classes are scanned when they are loaded by the JVM. Can you verify that the class that you are setting the breakpoint in is loaded at all?

**Comment by yuvaldrori** [comment]
Hi.

I get the error when setting a breakpoint but I see an error in the cdbg log in the Tomcat logs directory when the container starts about not finding the extra class path I added using cdbg_extra_class_path.
The workaround I found was to add the application to the webapps Tomcat directory already deployed and not as a war file and adding the classes directory to cdbg_extra_class_path. I could not make it work while adding the war file to the container.

Thanks!
